cves/CVE-2010-3718.yml

```diff
@@ -43,7 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  In Tomcat, web applications are normally allowed access to a particular work
+  directory. The applications are granted both read and write permissions so
+  they can use temporary files in that directory. Tomcat has an attribute that
+  dictates the location of that work directory. To prevent web applications from
+  changing the work directory, they should not be able to modify that attribute.
+  However, as a result of this vulnerability, web applications were able to
+  change that attribute, which meant that malicious web applications could read
+  and write to any area on the file system.
```

Comment:
On what file system?

Reply:
I updated it to say "the user's file system." Do you think this is descriptive enough?